.env.template

```diff
@@ -30,7 +30,11 @@ SESSION_ID_COOKIE_SECRET=my-cookie-secret-key
 USE_HTTPS=false
 
 # RStudio Server
-RSTUDIO_SERVER_URL=http://localhost:8787
+RSTUDIO_DOWNLOAD_SERVER_URL=
```

Comment:
since this is our domain, any other way we can get it dynamically ?

Reply:
this was to put the tunnel to localhost to download from RStudio, in production shouldn't be a problem.
now this URL is taken automatically with the request -> const serverUrl = Request.getServerUrl(req), but in local I had to put [ngrok](https://ngrok.com/) to be able to download the Code.

I can remove this config if you consider that it is cleaner

Reply:
add a comment, and add it to env.template

Reply:
done